yurt/fabfile/add.py

```diff
@@ -57,7 +53,18 @@
 
 
 @main.command()
-def remote_server():
+@click.option("--git_repo", default=None, help="Git Repo Link to Yurt project")
+@click.option("--env", default=None, help="Environment name (i.e. 'Development', 'Staging')")
+@click.option("--abbrev_env", default=None, help="Abbreviated env name (i.e. 'dev', 'stage')")
+@click.option("--app_host_ip", default=None, help="Public DNS of App Server")
+@click.option("--db_host_ip", default=None, help="IP of DB Server")
+@click.option("--debug", default=None, help="Runs debug mode (use `True` or `False`)")
+@click.option("--num_gunicorn_workers", default=None, help="Number of Gunicorn workers")
```

Comment:
we should have defaults for the gunicorn works and max requests if we don't because most people won't know what to fill out here